alembic/versions/20230512_5a425ebe026c_migrate_millenium_apis_to_post.py

```diff
@@ -81,7 +81,7 @@ def upgrade() -> None:
         if config_results and len(post_results) > 0:
             use_post = post_results[0][0]
             if use_post is None:
-                use_post == "false"
+                use_post = "false"
```

Comment:
This is the actual fix. The rest of this PR is just test setup and refactoring 😅